source/configure/calls-deployment.rst

```diff
@@ -380,3 +386,32 @@ Can the traffic between Mattermost and ``rtcd``  be kept internal or should it b
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When possible, it's recommended to keep communication between the Mattermost cluster and the dedicated ``rtcd`` service under the same private network as this can greatly simplify deployment and security. There's no requirement to expose ``rtcd``'s HTTP API to the public internet.
+
+Troubleshooting
+---------------
+
+Connectivity issues
+~~~~~~~~~~~~~~~~~~~
+
+If calls are failing to connect or timing out, it's likely the `RTC Server Port <plugins-configuration-settings.html#rtc-server-port-udp>`__ is not open or forwarded correctly. An easy way to check whether data can go through is to perform some tests using the ``netcat`` command line tool.
+
+On the host running Calls (could be the Mattermost instance itself or the one running ``rtcd`` depending on the chosen setup), run the following:
+
+.. code-block:: bash
+
+   nc -l -u -p 8443
+
+On the client side (i.e. the host where you'd normally access Mattermost from), run the following:
```

Comment:
This wording is a little confusing, using `host` to refer to the client machine. Maybe:
```suggestion
On the client side (i.e., the machine you would normally use to run the Mattermost desktop app or browser), run the following:
```